### PR TITLE
Support gym

### LINKED ISSUE #4794: Add Reinforcement learning Duet Notebooks
## Description
Add two notebook's which reflect the Reinforcement learning example split into DO (Data Owner) and DS (Data Scientist):
https://github.com/pytorch/examples/blob/master/reinforcement_learning/

## Definition of Done
The partially runnable Reinforcement learning notebook should be in the examples/duet/reinforcement_learning folder and a README.md should be added in the parent examples/duet directory with a link to the original example and our notebook.

### PR DESCRIPTION
**Pull request by Koukyosyumei**
## Description
support gym
closes #4794

add

            gym.make
            gym.wrappers.time_limit.TimeLimit.seed
            gym.wrappers.time_limit.TimeLimit.reset
            gym.wrappers.time_limit.TimeLimit.step

            torch.from_numpy
            torch.cat
            torch.stack
            torch.distributions.Categorical
            torch.distributions.Categorical.log_prob
            torch.distributions.Categorical.sample

notebooks (covered by integration tests)

            examples/duet/reinforcement_learning/Actor_Critic_Syft_Data_Owner.ipynb
            examples/duet/reinforcement_learning/Actor_Critic_Syft_Data_Scientist.ipynb
            examples/duet/reinforcement_learning/Reinforcement_Syft_Data_Owner.ipynb
            examples/duet/reinforcement_learning/Reinforcement_Syft_Data_Scientist.ipynb

## Affected Dependencies
gym

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

**Review by tudorcebere — APPROVED** [review]

**Comment by tudorcebere** [comment]
Awesome work @Koukyosyumei!

One small comment, try (at least from now on) to make meaningful commits/commit messages. (I can teach you how to use rebase on rewriting history if you don't want to keep everything tidy)
